docs/static/contrib-acceptance.asciidoc

```diff
@@ -15,5 +15,5 @@ If we list your plugin in the Logstash Reference, we point to your documentation
 * **Tests.** Your plugin must contain tests to be accepted. You can refer to http://betterspecs.org/ for examples. 
 ** Step 1. Enable travis on your account
 ** Step 2. Import our standard travis.yml https://github.com/logstash-plugins/.ci/blob/1.x/travis/travis.yml, as shown in the
-https://github.com/logstash-plugins/logstash-filter-fingerprint/blob/master/.travis.yml[fingerprint filter example].
+https://github.com/logstash-plugins/logstash-filter-fingerprint/blob/main/.travis.yml[fingerprint filter example].
```

Comment:
Should we change links that point to the `master` branch for plugins repos that have not been converted master->main yet? (Here's one example.) This will result in a lot of 404's. Fortunately, we shouldn't break the docs build in cases where we use the full path rather than using an attribute.

Reply:
indeed, my thinking is to execute the branch rename in all plugins and then do it in elastic/logstash, then apply this (and other) PRs

Reply:
So all plugin repos first, then `logstash` and `logstash-docs` repos? And then shortly thereafter the PR for  `elastic/docs`: https://github.com/elastic/docs/pull/2279?

Reply:
Yes. after all branches are renamed it's time to merge the PRs listed in https://github.com/elastic/logstash/issues/12052